src/FieldTrait.php

```
@@ -414,4 +414,142 @@ public function disableFormBrowserValidation(string $selector): void {
+    if ($element === NULL) {
+      $xpath = sprintf(
+        '//*[contains(text(), "%s")]/ancestor::div[contains(@class, "field--")]//input[contains(@name, "[%s][%s]")]',
+        $label,
+        $part,
+        $field
+      );
+      $element = $page->find('xpath', $xpath);
+    }
+
+    if ($element === NULL) {
+      throw new \Exception(sprintf('Datetime field "%s" with part "%s" and field "%s" not found.', $label, $part, $field));
```

PHPMD (2.15.0)

Missing class import via use statement (line '549', column '17'). (undefined)

(MissingImport)